spec/features/current_trips/admin_views_all_current_trips_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+feature "Admin sees all the current trips" do
+  describe "when click on the 'Current trips' button" do
+    scenario "successfully" do
+      school1 = create(:school, name: "St. Paul")
+      school2 = create(:school, name: "St. Peter")
+      build_stubbed(
+        :trip,
+        school_id: school1.id,
+        start_date: 2.month.ago,
+        end_date: 1.month.ago,
+      )
```

Comment:
We probably want to add this line here as well: `allow(Trip).to receive(:past).and_return([past_trip])`

Reply:
Done in `510c0d8`